gsa/src/gmp/models/__tests__/cpe.js

```diff
@@ -24,8 +24,8 @@ testModel(Cpe, 'cpe');
 
 describe('CPE model tests', () => {
   test('should parse severity correctly', () => {
-    const cpe = Cpe.fromElement({max_cvss: '5.0'});
-    const cpe2 = Cpe.fromElement({max_cvss: '10'});
+    const cpe = Cpe.fromElement({score: '50'});
+    const cpe2 = Cpe.fromElement({score: '100'});
 
     expect(cpe.max_cvss).toBeUndefined();
```

Comment:
```suggestion
    expect(cpe.score).toBeUndefined();
```

Reply:
This should now check if score is undefined. I think the reason for this line is to make sure that cpe.max_cvss (or rather cpe.score now) is removed after defining cpe.severity.